src/www/services_router_advertisements.php

```diff
@@ -420,10 +420,7 @@ function addRow() {
                       </div>
                       <br />
                       <input id="rasamednsasdhcp6" name="rasamednsasdhcp6" type="checkbox" value="yes" <?=!empty($pconfig['rasamednsasdhcp6']) ? "checked='checked'" : "";?> />
```

Comment:
I think we should move both checkboxes into a new row with "DNS options" ?

Their help text is correct after b504bc8 (dns settings includes search list and servers